.unreleased/pr_6992

```diff
@@ -0,0 +1,2 @@
+Fixes: #6975 Fix segfault in compress_chunk with primary space partition
+Thanks: @srieding for reporing issue with partially compressed chunks and ordering on joined columns
```

Comment:
```suggestion
Thanks: @srieding for reporting an issue with partially compressed chunks and ordering on joined columns
```

Reply:
I vaguely remember there being other issues with the same error message,

Reply:
https://github.com/timescale/timescaledb/issues/2232

Reply:
Yeah, but it doesn't look like it was caused due to this problem.